src/openrct2/interface/Cursors.cpp

```diff
@@ -14,39 +14,40 @@
 
 namespace Cursor
 {
-    uint8_t FromString(const std::string& s, uint8_t defaultValue)
+    uint8_t FromString(const std::string& s, CursorID defaultValue)
```

Comment:
why isn't this returning a CursorID?

Reply:
At first, because this was always assigned to a _legacyType that was expecting a uint8_t.
That's why I asked about changing their type too.
Anyway, after refactoring the rct_small_scenery_entry::tool_id (and others) to CursorID type, this is resolved :)